rust/spglib-sys/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "spglib-sys"
+version = "0.1.0"
+authors = ["Seaton Ullberg <seatonullberg@gmail.com>"]
+edition = "2018"
+license = "BSD-3-Clause"
+description = "Raw bindings to spglib"
+repository = "https://github.com/seatonullberg/spglib"
```

Comment:
This URL here too should point to this repository.

Reply:
Also fixed in https://github.com/atztogo/spglib/pull/89/commits/ec957715a6d3800a2a0a4a6bca6a489474b982c1